src/constants/translations/en/tutor-profile-page.json

```diff
@@ -20,6 +20,19 @@
   "videoPresentation": {
     "title": "Video Presentation"
   },
+  "aboutTutor": {
```

Comment:
There is no reason to add this text to translation, in future tutor will paste what he wants there

Reply:
> There is no reason to add this text to translation, in future tutor will paste what he wants there

Sure. but for now it a good place to pass text. When we'll work with this feature then we can replace logic. For now it's mocked text data.
Suggest leave it only inside in the component without translations ?